database/migrations/2014_10_12_000000_create_users_table.php

```diff
@@ -21,7 +21,7 @@ public function up()
             $table->string('user_address')->nullable();
             $table->string('google_id')->nullable();
             $table->string('password');
-            $table->string('user_gender')->nullable();
+            $table->enum('user_gender', ['Male', 'Female', 'Prefer not to say']);
```

Comment:
The default value should be "Prefer not to say" then, Please put a default value, otherwise, I think it will create an error when it generates a new user